src/Engine/Army.java

```diff
@@ -76,35 +76,32 @@ public void deInitForGame(GameInstance game)
       co.deInitForGame(game);
   }
 
-  /**
-   * Collect income and handle any COModifiers.
-   * @param map
-   */
-  public GameEventQueue initTurn(MapMaster map)
+  public GameEventQueue getAbilityRevertEvents(MapMaster map)
   {
     GameEventQueue events = new GameEventQueue();
 
-    money += getIncomePerTurn();
     for( Commander co : cos )
     {
-      events.addAll(co.initTurn(map));
+      events.addAll(co.getAbilityRevertEvents(map));
     }
```

Comment:
That's a bold move. I guess only one CO per army can have an ability active at once?

Reply:
No, but they should all deactivate at the same time.

Reply:
Or does persistent tags allow multiple actives at once?

Reply:
Team Merge does.
Persistent does not.